distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -21,9 +22,15 @@ Configure the CDM from the ${admin-console}:
 . Select the *Configuration* tab.
 . Select *Catalog Content Directory Monitor*.
 
+.Content Directory Monitor Permissions
+[IMPORTANT]
+====
+If changing the monitored directory directory from the default, make sure to set permissions on the new directory to allow ${branding} to access it.
```

Comment:
I don't think there is a 'default' in the monitored directory. I think we would want to mention that not only does the directory need to be accessible by the user running the ddf but that you will have to add the directory to the security manager policy if the directory is outside of the install directory.

Reply:
That's much clearer.